src/components/CertificateViewer.js

```diff
@@ -5,55 +5,80 @@ import { ErrorBoundary } from "./ErrorBoundary";
+import { getData } from "@govtechsg/open-attestation";
+import { Tab } from "react-bootstrap";
+import { AttachmentLink } from "./UI/AttachmentLink";
 
 export const CertificateViewer = (props) => {
   const { document } = props;
   const tokenRegistryAddress = getTokenRegistryAddress(document);
   const [templates, setTemplates] = useState([]);
   const [selectedTemplate, setSelectedTemplate] = useState("");
+  const originalData = getData(document);
+  const attachments = originalData?.attachments;
+  const hasAttachments = attachments && attachments.length > 0;
 
   const updateTemplates = useCallback((templates) => {
     setTemplates(templates);
     setSelectedTemplate(templates[0].id);
   }, []);
 
   const validCertificateContent = (
-    <>
-      <section className="bg-blue-lighter no-print">
+    <Tab.Container defaultActiveKey="tab-document">
+      <div className="bg-blue-lighter no-print">
         <DocumentStatus verificationStatus={props.verificationStatus} />
         {tokenRegistryAddress && <AssetManagementContainer document={document} />}
+        <MultiButtons tokenRegistryAddress={tokenRegistryAddress} />
         <MultiTabs
+          tokenRegistryAddress={tokenRegistryAddress}
+          hasAttachments={hasAttachments}
+          attachments={attachments}
           templates={templates}
+          setSelectedTemplate={setSelectedTemplate}
           selectedTemplate={selectedTemplate}
-          onSelectTemplate={(selectedTemplate) => setSelectedTemplate(selectedTemplate)}
-          tokenRegistryAddress={tokenRegistryAddress}
         />
-      </section>
-      <section>
-        <DocumentUtility className="no-print" document={document} handleSharingToggle={props.handleSharingToggle} />
-        <div className="bg-white py-3">
-          <DecentralisedRendererContainer
-            rawDocument={document}
-            updateTemplates={updateTemplates}
-            selectedTemplate={selectedTemplate}
-          />
-        </div>
-      </section>
+      </div>
+      <div className="bg-white">
+        <Tab.Content className="py-4">
+          <Tab.Pane eventKey="tab-document">
+            <DocumentUtility className="no-print" document={document} handleSharingToggle={props.handleSharingToggle} />
+            <DecentralisedRendererContainer
+              rawDocument={document}
+              updateTemplates={updateTemplates}
+              selectedTemplate={selectedTemplate}
+            />
+          </Tab.Pane>
+          {hasAttachments && (
+            <Tab.Pane eventKey="tab-attachment">
+              <div className="container-custom">
+                <div className="row">
+                  {attachments.map(({ filename, data, type }) => (
+                    <div className="col-6 col-lg-4 col-xl-3 mb-3" key={data}>
+                      <AttachmentLink filename={filename} data={data} type={type} />
+                    </div>
+                  ))}
+                </div>
+              </div>
```

Comment:
Possible to move this into a separate component so that it will be easier to search for this piece?

Something like `AttachmentTab`?